src/lib/kube_job.ml

```diff
@@ -16,11 +16,24 @@ module Specification = struct
     let point m = m.point
     let read_only m = m.read_only
   end
+  module File_contents_mount = struct
```

Comment:
This one is mount the contents as a string (which is indeed also a constant), so I called it like that.
I was keeping the option of later having `` `Constant`` mounts that come from files, not strings, but maybe overthinking...